### PR TITLE
refactor(value-list-item): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

File: src/components/calcite-value-list-item/calcite-value-list-item.scss [hunk 1]
```diff
@@ -20,25 +17,24 @@ calcite-pick-list-item {
...
+  @apply flex 
+    items-center 
+    justify-center 
+    py-0 
+    px-1 
+    bg-transparent 
+    border-none 
+    text-color-1
+    cursor-move;
   line-height: 0;
```
Comment (jrowlingson) [inline src/components/calcite-value-list-item/calcite-value-list-item.scss:29]: Would `leading-none` work here?

Comment (julio8a) [reply src/components/calcite-value-list-item/calcite-value-list-item.scss:29]: leading none sets it to 1, is that acceptable?

Comment (jrowlingson) [reply src/components/calcite-value-list-item/calcite-value-list-item.scss:29]: we may be able to remove it? setting the line-height to 0 is usually a hack

**Comment by julio8a** [comment]
this one is good to go! :)

**Review by jrowlingson — APPROVED** [review]

**Pull request merged**